fable-test-app/src/pages/ViewHolidaysNation.tsx

```diff
@@ -0,0 +1,131 @@
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+// Components (internal)
+import { DateModified, Heading, Select } from '../components';
+import { formatDate } from '../utils/utils';
+
+const ViewHolidaysNation = () => {
+  // State variables
+  const [holidays, setHolidays] = useState([]);
+  const [year, setYear] = useState<string>('2024');
```

Comment:
Could we use a computer current year instead of a hard coded value? Makes it so we don't have to touch this portion if we keep on using this in the next few years